client/app.jsx

```diff
@@ -37,36 +35,17 @@ if (!serverBootstrap) {
   const clientBootstrap = parseBootstrap(location.search);
 
   if (clientBootstrap) {
-    // Bootstrap.
-    flux.bootstrap(JSON.stringify({
-      ConvertStore: clientBootstrap
-    }));
-
-    // Set up listeners to render.
-    // Need the update conversions to **fully complete** to be consistent for
-    // the rendering.
-    deferRender = true;
-    const actions = flux.getActions("ConvertActions");
-    const listener = new ActionListeners(flux);
-    const _done = () => {
-      // Render
-      render();
-
-      // Remove all of the listeners for our custom action listener.
-      listener.removeAllActionListeners();
-    };
-    listener.addActionListener(actions.UPDATE_CONVERSIONS, _done);
-    listener.addActionListener(actions.CONVERSION_ERROR, _done);
-
-    // Trigger fetch.
-    actions.fetchConversions(
+    store.createStore(clientBootstrap);
+    store.dispatch(fetchConversions(
```

Comment:
QUESTION: Is this synchronous? Presumably _not_ because has to go to backend for values.

Which also means we _shouldn't_ render until we have data.

I think this means we need to redo the `let deferRender = false;` and deferred render scheme that has been removed in this file in this diff.